common/internal_model/src/main/scala/weco/catalogue/internal_model/index/WorksIndexConfig.scala

```diff
@@ -76,6 +76,9 @@ object WorksIndexConfig extends IndexConfigFields {
       val identifiersPath = List("search.identifiers")
       val titlesAndContributorsPath = List("search.titlesAndContributors")
 
+      val newIdentifiersPath = List("query.allIdentifiers")
```

Comment:
I've had to rename this because `query.identifiers` already exists – it's the list of all source identifiers on a work, to match the field on the display model.

(Previously we distinguished between source/other identifiers in the work model, and indexed both separately – this has simplified that.)